CHANGELOG.md

```diff
@@ -2,12 +2,27 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v1.2.0](https://github.com/voxpupuli/puppet-puppetwebhook/tree/v1.2.0) (2019-03-01)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-puppetwebhook/compare/v1.1.0...v1.2.0)
+
+### Added
+
+- Add `pkg\_version` parameter [\#16](https://github.com/voxpupuli/puppet-puppetwebhook/pull/16) ([alexjfisher](https://github.com/alexjfisher))
+
+### UNCATEGORIZED PRS; GO LABEL THEM
+
+- Remove broken default `command\_prefix` [\#18](https://github.com/voxpupuli/puppet-puppetwebhook/pull/18) ([alexjfisher](https://github.com/alexjfisher))
+- Fix service not restarting on package update [\#17](https://github.com/voxpupuli/puppet-puppetwebhook/pull/17) ([alexjfisher](https://github.com/alexjfisher))
+- Fix `source` in metadata.json [\#13](https://github.com/voxpupuli/puppet-puppetwebhook/pull/13) ([alexjfisher](https://github.com/alexjfisher))
+
 ## [v1.1.0](https://github.com/voxpupuli/puppet-puppetwebhook/tree/v1.1.0) (2019-01-17)
 
 [Full Changelog](https://github.com/voxpupuli/puppet-puppetwebhook/compare/v1.0.0...v1.1.0)
 
-### Fixed
+### UNCATEGORIZED PRS; GO LABEL THEM
 
+- Release 1.1.0 [\#12](https://github.com/voxpupuli/puppet-puppetwebhook/pull/12) ([dhollinger](https://github.com/dhollinger))
```

Comment:
This is labeled 'skip-changelog', but the Rakefile is looking for 'maintenance'.